server/utilis/validation/signinValidation.js

```diff
@@ -5,7 +5,7 @@ const signinValidation = async (req, res, next) => {
   const { email, password } = req.body;
   const schema = Joi.object({
     email: Joi.string().email().required(),
-    password: Joi.string().min(6).required()
+    password: Joi.string().min(8).required()
```

Comment:
sorry for missing this, but the minimum in signup is 6, so please discuss with @khamis-hesham if you want 6 or 8 as a minimum and modify front-end and backend validation with the agreed value